voice/bxml/verbs/gather.md

```diff
@@ -4,17 +4,17 @@ The Gather verb is used to collect digits for some period of time.
+| Attribute         | Description                                                                                                                                                                                                                                                                                                                                     |
+|:------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| gatherUrl         | (optional) URL to send [Gather event](../callbacks/gather.md) to and request new BXML.                                                                                                                                                                                                                                                          |
+| gatherMethod      | (optional) The HTTP method to use for the request to `gatherUrl`. GET or POST. Default value is POST.                                                                                                                                                                                                                                           |
+| username          | (optional) The username to send in the HTTP request to `gatherUrl`.                                                                                                                                                                                                                                                                             |
+| password          | (optional) The password to send in the HTTP request to `gatherUrl`.                                                                                                                                                                                                                                                                             |
+| tag               | (optional) A custom string that will be sent with this and all future callbacks unless overwritten by a future `tag` attribute or cleared.<br><br>May be cleared by setting `tag=""`<br><br>Max length 256 characters.                                                                                                                          |
+| terminatingDigits | (optional) When any of these digits are pressed, it will terminate the Gather. Default behavior is disabled. This feature can also be explicitly disabled with `""`.                                                                                                                                                                            |
```

Comment:
I would just say "Default: none (`""`)"

Reply:
I had typed up a reply, but I guess I didn't save it.
Basically, the established convention is `Default value is $VALUE`, and I'd like to be consistent.
Should I change all of the existing default messages?

Reply:
I don't understand your point. The message here doesn't say `Default value is $VALUE`. What I suggested is closer to that I think.

Reply:
Hmmm. How about this?
```
... Default value is `""`, which disables the timeout.
```

Reply:
Sure